packages/web-components/@sfx/ui/stories/index.ts

```diff
@@ -0,0 +1,36 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import '../src/index';
+
+const items = [
+  {
+    label: 'Pen'
+  },
+  {
+    label: 'Headphones'
+  },
+  {
+    label: 'Duck'
+  },
+  {
+    label: 'Chocolate'
+  }
+];
+
+storiesOf('UI|List', module)
+  .addDecorator(withKnobs)
+  .add(
+    'Default',
+    () => `
+      <sfx-list items="${text('List Items', JSON.stringify(items))}"></sfx-list>
+    `
+  )
+  .add(
+    'With Title',
+    () => `
+      <sfx-list sfxlisttitle="${text('Sfx List Title', 'Items')}" items="${text(
```

Comment:
Instead of breaking up the function call, I would break up the attributes instead, one on each line.